### PR TITLE
adding the ability to export or not the subnet routes with public ip

### PR DESCRIPTION
**Pull request by itManuel**
I added the ability to avoid the exchange subnet routes with public IPv4, adding a variable. The default value will allow the module to work as until now (the default value is TRUE so if you want to avoid that you should change it to FALSE)

Regards,

### CODE REVIEW

File: modules/net-vpc-peering/variables.tf [hunk 1]
```diff
@@ -27,6 +27,12 @@ variable "export_peer_custom_routes" {
   default     = false
 }
 
+variable "export_subnet_routes_with_public_ip" {
```
Comment (ludoo) [inline modules/net-vpc-peering/variables.tf:30]: can we rename to `export_public_ip_routes` for consistency?

**Review by ludoo — CHANGES_REQUESTED** [review]
Dropped a minor comment, can you make sure CLA and checks pass? If you are unsure on how to do that just ping us here. Thanks for this!

File: modules/net-vpc-peering/variables.tf (hunk 1 above)
Comment (itManuel) [reply modules/net-vpc-peering/variables.tf]: Sure, please check again

Comment (itManuel) [reply modules/net-vpc-peering/variables.tf]: I singned the CLA, which checks are you talking about?

**Comment by ludoo** [comment]
hmmm
image

**Comment by ludoo** [comment]
if both users are you, ping me internally :)

**Comment by itManuel** [comment]
can you cancel this? I'll try to do it again BUT without multi-personality issues.

**Pull request closed without merging**